README.Rmd

```diff
@@ -63,83 +63,80 @@ devtools::install_github("bcgov/bcgovr")
+
+**Step 1: Set up a [remote repository](https://help.github.com/articles/about-remote-repositories/)**
+
+For a remote in [github.com/bcgov](github.com/bcgov), click on the green **New** button to create a new repository. [Choose a repository name](https://github.com/bcgov/BC-Policy-Framework-For-GitHub/blob/master/BC-Gov-Org-HowTo/Naming-Repos.md)---our example repository name is `bcgovr_analysis`. You can open an empty repository---without initializing a README, a .gitignore file or a license--- as `bcgovr` will take care of all of that for you later. Copy the URL of the repository, you'll need it to set up your local repository in the next step.
+
+
+**Step 2: Set up & populate local repository using `bcgovr`**
+
+*RStudio GUI users:* Open a fresh session of RStudio where you will open your New Project. By default, RStudio will create any New Project in the current working directory. You can check the working directory using `getwd()`. Set your preferred local working directory in the R console (e.g. `setwd(C:/_dev)`) or using the RStudio navigation tools (Session -> Set Working Directory -> Choose Directory...). Now open your New Project: Select Version Control from the Create Project menu -> Select Git -> Paste the URL of the github.com/bcgov repository from step 1 in Repository URL -> Click 'Create Project'. The local project name will be inherited from the remote repository, and your local project is now "connected" to your remote in [github.com/bcgov](github.com/bcgov).
+
+You can now run `analysis_skeleton()` to create a 'ready-to-go' set of folders & files in your local repository that encourage best practice in scientific computing and include all of the [required bcgov items](https://github.com/bcgov/BC-Policy-Framework-For-GitHub/blob/master/BC-Gov-Org-HowTo/Cheatsheet.md).
+
 
 ```{r, eval = FALSE}
-bcgovr::analysis_skeleton() 
+bcgovr::analysis_skeleton() ## directly calls the function from `bcgovr` library
+## OR
+library(bcgovr)
+analysis_skeleton()
 ```
 
-Frequently you may want to create an analysis skeleton outside of the working directory. This directory can be set using the `path` argument. Consider if you wanted to create an analysis skeleton named `bcgovr_analysis` in the `C:/_dev` directory. In this case, you would directly input the path which would also automatically create a `bcgovr_analysis` folder if it did not exist already:
+
+*R/RStudio Console Users:* You can set your local working directory in the R/RStudio Console (e.g. `setwd(C:/_dev)`) or you can use the `path` argument in `analysis_skeleton` to set the path where you want your local repository to exist. Supply the remote location using the `clone_git` argument.
```

Comment:
What about something like: 

'As an alternative to using the 'New Project -> Version Control...' dialogue in RStudio, you can specify the url of the remote repository using the `git_clone` argument in `analysis_skeleton`. Be sure to either specify your new analysis directory using the `path` argument, or `setwd("c:/my_bcgov_analysis")` before running `analysis_skeleton()`'